### ISSUE TITLE
Compile error: cannot specify -o with -c, -S or -E with multiple files gcc

### ISSUE THREAD

**Issue by Mihaylov93** [SimulIDE/SimulIDE#13]
Compilation stops with the given error.

Environment: 

- Windows 10 64 bits
- MinGW 64 bits
- Qt 5.15.0

**Comment by Foadsf** [comment]
@Mihaylov93 would you please add the complete error message too?

**Comment by Foadsf** [comment]
I would also recommend we try the MSYS2 environment rather than the vanilla MinGW.

**Comment by Mihaylov93** [comment]
That is the complete error.

```
\Qt\5.15.0\mingw81_64\include\QtCore -Idebug -I/include -IE:\Qt\5.15.0\mingw81_64\mkspecs\win32-g++  -o debug\*.o ../src/gpsim/registers/*.cc
g++: fatal error: cannot specify -o with -c, -S or -E with multiple files
compilation terminated.
mingw32-make[1]: *** [Makefile.Debug:522: debug/*.o] Error 1
mingw32-make: *** [Makefile:45: debug] Error 2
```

The issue is the source and header files with the asterisk * although qmake allows that it is bad practice.

I already have a workaround.

**Comment by Mihaylov93** [comment]
Check this topic: https://stackoverflow.com/a/16439457

QMake also didn´t like the recursive solutions mentioned there. One solution is to properly expand the * in the paths.

Easily doable with bash;
`find . -name '*.cpp' -exec echo .{} \\ \;`
`find . -name '*.h' -exec echo .{} \\ \;`

Don´t forget to remove the last \


I expanded the paths in [5c15a89ab2ff96d9567e486dc10820fbe5f57044](https://github.com/SimulIDE/SimulIDE/pull/15/commits/5c15a89ab2ff96d9567e486dc10820fbe5f57044)

**Comment by Mihaylov93** [comment]
Resolved by #15
